exporter/zipkinexporter/zipkin.go

```diff
@@ -104,8 +113,14 @@ func createZipkinExporter(logger *zap.Logger, config configmodels.Exporter) (*zi
 
 func (ze *zipkinExporter) PushTraceData(ctx context.Context, td consumerdata.TraceData) (droppedSpans int, err error) {
 	tbatch := []*zipkinmodel.SpanModel{}
+
+	var resource *resourcepb.Resource = nil
```

Comment:
nit: in go typically we don't assign the zero value, so 
```suggestion
	var resource *resourcepb.Resource
```